src/gopher.cc

```diff
@@ -686,7 +686,7 @@ gopherToHTML(GopherStateData * gopherState, char *inbuf, int len)
             }
 
             }           /* HTML_CSO_RESULT */
-
+        break;
```

Comment:
Same as another change request: Please move this `break` into the case`{}` scope. Doing so will improve code formatting as well.

Reply:
ok